bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -374,6 +374,17 @@ bool PreProcessor::validateSubpoolsConfig() {
   return true;
 }
 
+const concordUtil::MultiSizeBufferPool::SubpoolsConfig PreProcessor::defaultSubpoolsConfig_ = {{65536, 0, 1400},
```

Comment:
- here we dont need the explciit numbers, use shift:
e.g (with int, choose any suitable type)
`constexper int MB = (1 << 20);`
so for 1st raw:
{{65 * MB, 0, 1400}

- in below line you used 140 and not 1400
- try defining it as static constexpr  concordUtil::MultiSizeBufferPool::SubpoolsConfig and see if it's compile. if not, just use static const.

Reply:
OK.
static constexpr is not compiled.